### DISCUSSION TITLE
Apache 2 Debian Default page

### DISCUSSION THREAD

**Discussion by Simmy-Simalia** [kelinger/OmniStream#62] [General]
Hello,

I'm not new to the scene but I'm still lacking some comprehension hoping you can help.

I have loaded Omnistream, launched containers, DNS records are correct. However when I navigate to the domains e.g., deluge.domain.com I am greeted with the Apache 2 Debian Default page because I'm being directed to domain.com:80. Have I installed this wrong or what am I missing.

Thanks,
Simalia

**Comment by DizFreak13** [comment]
Hey! Just came here to say that I'm experiencing this same issue! Was hoping someone answered this, but since it's been a while, have you been able to figure out what happened?

**Comment by robnester** [comment]
Check to ensure that you have traefik enabled and launched as that's going to be serving as your [reverse proxy](https://www.cloudflare.com/learning/cdn/glossary/reverse-proxy/) to other services, including apache. If you've just enabled apache, or you've enabled apache, as well as other services, and not traefik you would, most likely, see traffic hitting your apache instance for all of your subdomains
